data api/src/main/java/de/tinf15b4/kino/api/KinoWebDataService.java

```diff
@@ -106,12 +104,14 @@ private static void jettyEnableInherit(Server server) {
         }
     }
 
-    @Bean
-    public CommandLineRunner loadData(DataInitializer initializer) {
-        return (args) -> {
-            initializer.initialize();
-        };
-    }
+    // DON'T enable those lines if you use an existing database. The fake initializer will cause exceptions when writing
+    // to a filled database
+    // @Bean
+    // public CommandLineRunner loadData(DataInitializer initializer) {
+    // return (args) -> {
+    // initializer.initialize();
+    // };
+    // }
```

Comment:
We could conditionally enable this by checking the `spring.jpa.hibernate.ddl-auto` property then there is just one place to change if we want the fake data